iron-request.html

```diff
@@ -229,12 +229,21 @@
         }, this);
       }
 
+      var contentType;
+      if (options.headers) {
+        contentType = options.headers['Content-Type'];
+      }
+      body = this._encodeBodyObject(options.body, contentType);
```

Comment:
Oh good catch.

My preference for catching this class of bug is strict mode. Added 'use strict' to both iron-ajax and iron-request and confirmed that this bug was caught.